src/components/tabs/Company.svelte

```diff
@@ -1,32 +1,97 @@
 <script lang="ts">
-	import { Accordion, AccordionItem, Input, Label, NumberInput, Select, Textarea } from "flowbite-svelte";
+	import { Accordion, AccordionItem, Card, Checkbox, Input, Label, NumberInput, Select, Textarea } from "flowbite-svelte";
 	import { store } from "$store/store";
 
-	type SelectEventWithTarget = Event & { currentTarget: HTMLSelectElement };
+	type InputEventWithTarget = Event & { currentTarget: HTMLInputElement };
 	type TrueOrFalse = "true" | "false";
```

Comment:
It would be better if I could eliminate `TrueOrFalse` and use plain `boolean` instead.
It's ugly and I've used type assertions repeatedly.